planet4-gutenberg-blocks.php

```diff
@@ -439,6 +439,44 @@ function ( $std_class_object ) {
 	],
 ];
 
+add_filter(
+	'render_block_core/query-pagination-previous',
+	function ( $content, $parsed, $block ) {
+		if ( empty( $block->attributes['label'] ) ) {
```

Comment:
I just realized this could lead to an infinite loop if somebody saves `Prev` translation as an empty string :grin: , so you could replace this condition with `if ( ! array_key_exists( 'label', $block->attributes ) ) {`